pkgs/servers/web-apps/shiny-server/node-packages.nix

```diff
@@ -0,0 +1,1878 @@
+# This file has been generated by node2nix 1.4.0. Do not edit!
```

Comment:
This is a tough part about this PR. This is quite large for a single package -- 2,200 lines! Other than this, this seems okay. @svanderburg what is the feeling on these being in nixpkgs?

Reply:
I understand that this may be an issue. For what it's worth, there are other instances of the same problem:

orbekk@aji ~/b/nixpkgs (shiny)> find pkgs -name node-packages.nix | xargs ls -lh
-rw-r--r-- 1 orbekk users  27K Nov 24 17:43 pkgs/applications/editors/zed/node-packages.nix
-rw-r--r-- 1 orbekk users 111K Nov 24 17:43 pkgs/development/web/remarkjs/node-packages.nix
-rw-r--r-- 1 orbekk users  444 Nov 24 17:43 pkgs/misc/base16-builder/node-packages.nix
-rw-r--r-- 1 orbekk users 154K May 19  2017 pkgs/servers/web-apps/pump.io/node-packages.nix
-rw-r--r-- 1 orbekk users  60K Dec  7 19:59 pkgs/servers/web-apps/shiny-server/node-packages.nix
-rw-r--r-- 1 orbekk users  28K Nov 24 17:43 pkgs/tools/networking/airfield/node-packages.nix
-rw-r--r-- 1 orbekk users 3.0K May 19  2017 pkgs/tools/package-management/nixui/node-packages.nix